astropy/wcs/wcsapi/high_level_wcs_wrapper.py

```diff
@@ -69,3 +69,10 @@ def axis_correlation_matrix(self):
         See `~astropy.wcs.wcsapi.BaseLowLevelWCS.axis_correlation_matrix`
         """
         return self.low_level_wcs.axis_correlation_matrix
+
+    @property
```

Comment:
I think the best thing to do is to make this a method which calls the method on the low level object.
```suggestion
```